core/src/types.rs

```diff
@@ -172,6 +186,134 @@ pub enum TypedValue {
     Uuid(Uuid),                        // It's only 128 bits, so this should be acceptable to clone.
 }
 
+/// The values stored in a `StructuredMap` can be:
```

Comment:
The comment refers to `StructuredMap`, but it's attached to `Binding`.  Huh?

Reply:
This comment is cleaned up later.